### PR TITLE
fix: :rocket: Typo

### LINKED ISSUE #88: [Bug]: Typo in `restrictedWords.js`
### Describe the bug

`Classroom monitor` is written as `Classroon monitor`

### To Reproduce

1. Go to `Commands/restrictedWords.js`
2. Check line no. 7

### Expected Behavior

It should be `Classroom monitor`

### Screenshot/ Video

image

### Desktop (please complete the following information)

_No response_

### Additional context

I am opening a pull request for this soon

### PR DESCRIPTION
**Pull request by guptasajal411**
## Fixes Issue

Closes #88 

Fixed `Classroon monitor`

Here comes all the changes proposed through this PR

## Check List (Check all the boxes which are applicable) 

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots 
image
image

### CODE REVIEW

File: Commands/restrictedWords.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const warnEmbed = new Discord.MessageEmbed()
     .setTitle('Community monitor warning you!!')
     .setURL('https://github.com/kaiwalyakoparkar/classroom-monitor-bot')
     .setAuthor(
-        'Classroon monitor',
+        'Classroom monitor',
```
Comment (kaiwalyakoparkar) [inline Commands/restrictedWords.js:7]: Could you please capitalize "M" of monitor?

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

File: Commands/restrictedWords.js (hunk 1 above)
Comment (guptasajal411) [reply Commands/restrictedWords.js:7]: Sure

**Comment by guptasajal411** [comment]
Fixed `Classroon Monitor`

**Comment by kaiwalyakoparkar** [comment]
Could you please resolve the conflict?

**Comment by guptasajal411** [comment]
Sure, it's resolved now. I think we're good to go!

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket:

**Pull request merged**